app/controllers/orders_controller.rb

```diff
@@ -4,7 +4,7 @@ class OrdersController < ApplicationController
   before_action :authenticate_employee!
 
   def index
-    @orders = Order.where(employee: current_employee)
+    @orders = OrderSearch.new(params).results.where(employee: current_employee)
```

Comment:
@oskarlakner is there better way to get these results with current_employee?

Reply:
probably with Pundit's policy_scope, but it's ok what you have :)